doc/man1/flux-mini.rst

```diff
@@ -174,6 +174,15 @@ following additional job parameters:
    option is minutes when no units are otherwise specified. If the time
    limit is unspecified, the job is subject to the system default time limit.
 
+**--flags=FLAGS**
+   Set comma separated list of job submission flags.  The possible flags are
+   ``waitable``, ``novalidate``, and ``debug``.  The ``waitable`` flag will
+   allow the job to be waited on via ``flux job wait`` and similar API calls.
```

Comment:
Should note that `waitable` is instance owner only. Maybe "both `waitable` and `novalidate` require instance owner privileges"